jib-core/src/main/java/com/google/cloud/tools/jib/configuration/BuildContext.java

```diff
@@ -498,6 +520,15 @@ public boolean getAlwaysCacheBaseImage() {
     return layerConfigurations;
   }
 
+  /**
+   * Gets the registry mirrors.
+   *
+   * @return the registry mirrors
+   */
+  public ImmutableMap<String, ImmutableList<String>> getRegistryMirrors() {
```

Comment:
How about using `ImmutableListMultimap` instead of  `ImmutableMap<String, ImmutableList<String>>` here?

Reply:
Nice, I used it in a few places, and looks better.